### ISSUE TITLE
[PM, BE, FE, Design] 10-10EZ - SIGI: Perform team QA and Accessibility testing

### ISSUE THREAD

**Issue by hdjustice** [department-of-veterans-affairs/va.gov-team#49093]
**BLOCKED** due to pending decisions on content with stakeholders

## Background
Now that Self-Identified Gender Identity question is in Staging, we can begin our own QA and Accessibility QA on the page and content.

- [Use Cases](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/staging-test-accounts-Self-identifying-Gender-Identity.md) for QA
- [Testing for Accessibility](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/application/va-application/Self%20Identifying%20Gender%20Identity%20(SIGI)/QA/SIGI%20-%20Accessibility%20QA.md)

## Tasks
- [ ] Perform QA on the SIGI page, using the Use cases as a guide
- [ ] Perform Accessibility QA, using the test cases as a guide
- [ ] Any issues or discrepancies found will be documented in this ticket, and tag @markfallows 

## Acceptance Criteria
- All tests pass successfully without issues, errors or violations of accessibility guidelines

**Comment by hdjustice** [comment]
This issue is blocked due to pending decisions on Content between VA.gov and Stakeholders.  We will proceed with Accessibility Testing and Staging review once the decision is made and updates are complete.

**Comment by hdjustice** [comment]
Closing all SIGI tickets - There has been no further movement or discussion on this topic
